logger/logger_test.go

```diff
@@ -98,3 +118,68 @@ func TestError(t *testing.T) {
 		assert.Equal(t, tc.output, output, fmt.Sprintf("%s: expected %s got %s", desc, tc.output, output))
 	}
 }
+
+func TestLogLevel(t *testing.T) {
+	debugCases := map[string]struct {
+		input  string
+		output logMsg
+	}{
+		"debug log ordinary string": {"input_string", logMsg{"", ""}},
+		"debug log empty string":    {"", logMsg{"", ""}},
+	}
```

Comment:
You should move these into `TestDebug`. Just create new logger instance in for loop and set log level from test case. Same for other table maps in this function.